packages/desktop/src/renderer/components/ContextMenu/menus/ChannelContextMenu.container.tsx

```diff
@@ -10,10 +10,12 @@ import { ContextMenuItemProps } from '../ContextMenu.types'
 
 import { useModal } from '../../../containers/hooks'
 import { ModalName } from '../../../sagas/modals/modals.types'
+import { exportChats } from 'packages/desktop/src/utils/functions/exportChats'
```

Comment:
Please fix this import, this is the reason why all e2e tests and one desktop test failed.

Reply:
Sure thing! I'll do this right away.